connect/src/main/java/com/stripe/android/connect/AccountOnboardingView.kt

```diff
@@ -6,50 +6,51 @@ import android.util.AttributeSet
 import androidx.annotation.RestrictTo
 import androidx.core.content.withStyledAttributes
 import com.stripe.android.connect.webview.StripeConnectWebViewContainer
-import com.stripe.android.connect.webview.StripeConnectWebViewContainerImpl
 import com.stripe.android.connect.webview.serialization.SetOnExit
 import com.stripe.android.connect.webview.serialization.SetterFunctionCalledMessage
 import dev.drewhamilton.poko.Poko
 import kotlinx.parcelize.Parcelize
 
 @PrivateBetaConnectSDK
-class AccountOnboardingView private constructor(
+internal class AccountOnboardingView internal constructor(
```

Comment:
👀 Component views are `internal` while we focus on component workflows. In the future:
 * the component classes will be made public
 * this constructor will remain `internal`
 * the secondary constructor below will remain public for use in XML layouts.